workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/SpatialOSBuildConfigurationEditor.cs

```diff
@@ -320,48 +301,34 @@ private static string EnumFlagToString<TEnum>(TEnum value, Func<TEnum, string> n
                 enumNonZeroValues
                     .Where(enumValue =>
                         (sourceBitValue & enumValue.ToInt32(NumberFormatInfo.CurrentInfo)) != 0)
-                    .Select(nameFunction).ToArray());
+                    .Select(enumValue => enumValue.ToString(CultureInfo.InvariantCulture)).ToArray());
         }
 
         private void ConfigureBuildPlatforms(BuildEnvironmentConfig environmentConfiguration)
         {
             using (IndentLevelScope(1))
             {
-                var buildPlatformsString = EnumFlagToString(environmentConfiguration.BuildPlatforms,
-                    BuildPlatformToString);
-
                 EditorGUI.BeginChangeCheck();
-
+                
+                var buildPlatformsString = EnumFlagToString(environmentConfiguration.BuildPlatforms);
```

Comment:
Why do we not use `BuildPlatformToString` here any more? It was helping with the meaning of "Current"

Reply:
I had removed "Current" inbetween as we now display the current platform twice.. but couldn't find a good way around it without losing some context. will add it back